### ISSUE TITLE
Glowgrass missing? [1.15.2]

### ISSUE THREAD

**Issue by Qsandas** [TelepathicGrunt/UltraAmplifiedMod#12]
Looks like you don't get many bug reports. That's a good thing. But sadly I'm here to add one more. I'm not sure why, but I keep getting this error and crash when I try and launch Minecraft. There's a couple things that set this apart from normal crashes I've received over the years. 
1. I was playing with this mod just two days ago and I haven't done anything to change any of my mods between then and now.
2. I copy my custom pack over to my brother's computer, and there's no error for him, as I just checked 5 minutes ago.

[crash-2020-04-18_22.34.03-client.txt](https://github.com/TelepathicGrunt/UltraAmplifiedMod/files/4498431/crash-2020-04-18_22.34.03-client.txt)

Edit: It's possible this may be a twitch-induced problem, but I think it's still best that I reported it anyways

**Comment by TelepathicGrunt** [comment]
That's really odd as the glowgrass block should exist by the time it goes to register the rendering for the block. It works for me, your brother, and no one else reported this crash too which make this even more odd.

Though I still made this test version that delays the glowgrass render layer stuff even more. can you test this jar and see if this fixes the problem for you? I hope it does!
[ultra_amplified_dimension-1.15.2-7.3.3b.zip](https://github.com/TelepathicGrunt/UltraAmplifiedMod/files/4500250/ultra_amplified_dimension-1.15.2-7.3.3b.zip)

**Comment by Qsandas** [comment]
Ok, it turns out my config files got messed up somehow, and not just for this mod. I tried disabling the mod to see if the game would run and it gave me the exact same error on a different mod for a different item. Disabled that one, same story. Luckily I had a copy of my config files from when I shared them with my brother, and replaced my current ones (even though they should have been identical) and it finally solved the problem.
